source/dicom/valuerep.py

```diff
@@ -47,7 +47,17 @@ class DSfloat(float):
 
     """
     __slots__ = 'original_string'
-
+    def __getstate__(self):
```

Comment:
Spot on.  Thanks for this.  I have added some pickling tests and updated the get_state method to look for this.  It is strange that __slots__ is allowed to be either a string or an iterable, but it is.